app/models/invoice.rb

```diff
@@ -13,8 +13,14 @@ class Invoice < ApplicationRecord
     "completed": 2
   }
 
+  #class methods
```

Comment:
Cory looks like your going to have to teach me to query. These are very impressive.